### ISSUE TITLE
No hint path defined for [googlitics]

### ISSUE THREAD

**Issue by sunscreem** [bradcornford/Googlitics#3]
Hello - I've been trying to get this to work but when I call the analytics::render method I get:

```InvalidArgumentException in FileViewFinder.php line 112:
No hint path defined for [googlitics].```

I'm using 5.2 if that makes any difference?

**Comment by bradcornford** [comment]
The only thing i can see is that you would need to use:

```
Analytics::render()
```

**Comment by bradcornford** [comment]
Also ensure that you run the following commands:

```
php artisan vendor:publish --provider="Cornford\\Googlitics\\AnalyticsServiceProvider"

```
or

```
php artisan vendor:publish --tag=googlitics
```

**Comment by sunscreem** [comment]
Thanks Brad. Service provider and facade are all okay. And the config file is all set up.. but still no joy.

Here's my controller I'm using for testing. I'm maybe missing something obvious?

```<?php

namespace App\Http\Controllers;

use Analytics;
// use App\ActivityLog;
// use App\Http\Requests;
// use Illuminate\Http\Request;
use App\Http\Controllers\Controller;

class ActivityLogController extends Controller
{
    
    public function show()
    {
    	
		Analytics::trackPage('Homepage', 'Homepage Title');
		
		dd(Analytics::render());
    	
    	//$activityLog = ActivityLog::all();
   		// dd(ActivityLog::find(1)->user);
    	//return view('ajax/activityLog',compact('activityLog'));
    }
}
```

Which returns:

![screenshot-xyz app 2016-01-25 12-56-39](https://cloud.githubusercontent.com/assets/7523595/12551533/179dbff0-c363-11e5-8cff-9ea5dd5bdf44.png)

**Comment by bradcornford** [comment]
Can you try adding the below to you class "ActivityLogController":

```
use Cornford\Googlitics\AnalyticsServiceProvider as Analytics;

```

**Comment by sunscreem** [comment]
Sure..

It now returns 

```Fatal error: Call to undefined method Cornford\Googlitics\AnalyticsServiceProvider::trackPage()```

**Comment by bradcornford** [comment]
Are you sure you added the line above? Looks like you linked to the service provider instead of the facade.

**Comment by sunscreem** [comment]
That's what I was thinking too but .. I've been over everything a couple of time. I'm *must* be being dumb and missing something obvious. I've added you to the project Brad. I'm thinking you might be able to see where I've been stupid?

**Comment by bradcornford** [comment]
I've looked into the Service Provider and i made a mistake in defining the hint path for the views. I have now corrected this, and re-versioned 2.1.0. If you do a composer update you should be good to go. Sorry for the inconvenience!

**Comment by sunscreem** [comment]
Sorted :)

Composer update wasn't picking up the new release yet, but I've made the change manually and its outputing the GA code correctly now.

Thank you Sir!
